### PR TITLE
Python cleanups/v3

### PR DESCRIPTION
**Pull request by inashivb**
Previous PR: https://github.com/OISF/suricata/pull/9081

Changes since v2:
- update running instructions

For commit 120479a0a6bf996e2e389ea8047f469c31f0b2e3:
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/3432

We no longer use distutils as stated in the bug but this shebang is anyway more portable.

### CODE REVIEW

File: python/suricata/ctl/main.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# Copyright (C) 2018 Open Information Security Foundation
+#!/usr/bin/env python
+# Copyright (C) 2018-2023 Open Information Security Foundation
```
Comment (jasonish) [inline python/suricata/ctl/main.py:2]: As this file is never intended to be directly executed, I wonder if we should have the shebang here..

File: python/suricata/sc/suricatasc.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/python
-# Copyright(C) 2012-2020 Open Information Security Foundation
+#!/usr/bin/env python
+# Copyright(C) 2012-2023 Open Information Security Foundation
```
Comment (jasonish) [inline python/suricata/sc/suricatasc.py:2]: Same here.

Looks like these are both executable in current install, which is probably a mistake as well.

**Review by victorjulien — CHANGES_REQUESTED** [review]
Needs rebase.

File: python/suricata/ctl/main.py (hunk 1 above)
Comment (inashivb) [reply python/suricata/ctl/main.py:2]: You're right. It doesn't make sense. Should remove it..

File: python/suricata/sc/suricatasc.py (hunk 2 above)
Comment (inashivb) [reply python/suricata/sc/suricatasc.py:2]: Sounds fair. I have removed the x bit in the install in the newer version of this PR, please check.

**Pull request closed without merging**